### ISSUE TITLE
update error handeling on uplaod_edi_pacakge 

### ISSUE THREAD

**Issue by ErinCain** [CVPIA-OSC/EDIutils#161]
update_edi_pacakge - successfully uploaded a package but we got a response "timed out"

**Comment by alvizek** [comment]
Ashley, Erin, Liz paired on this and added a package size argument to api functions to increase sys.sleep time. This helped with some of the issues. The remaining issue is that for update_edi_package you get the `"Request timed out, check that you inputs are all valid, rerun evalutate_edi_package(), and try again"` error but the package uploaded. Need to try increasing max_iter higher or have 15 be the min sleep time. 

Next steps:
- Test higher max_iter
- Other options: min sleep is 15, or have two diff sleep time options (one for the iter chunk)
- update documentation to include package_size argument

**Comment by alvizek** [comment]
I did some testing on this and the results are not satisfactory though maybe OK for the short time as we will only be using the update function.

- increased `max_iter` for `update_edi_package` to 10. update successful on two attempts (meaning the appropriate message displayed and the package updated on EDI)
- `upload_edi_package` not successful. on every attempt `Request timed out, check that you inputs are all valid, rerun evalutate_edi_package(), and try again`. these are the changes i tried: package_size = "large", max_iter = 20, max_iter = 15.

**Comment by liz-fw** [comment]
I made 2 changes that now seem to have fixed the error in the `upload` function.

1. I hard coded the "scope" argument to `httr::GET `. The way the strsplit was coded, it was pulling the whole eml_file_path (so "data-raw/manual_testing_materials" etc.) and this was causing the `check_upload` function to always return `404`. There is probably a smarter way to do this but it should work for now
2. I changed the if statement to be `iter > max_iter` instead of the other way around. 

I also changed the default package size (so sleep_time) to be "small"  bc I think now it should hopefully work more quickly.
